### ISSUE TITLE
AVT 1 Skeleton: Search has a DAP violation for missing unique search label and label w/ descriptive text

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4179]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React skeleton for Search
Run DAP and the following issue is found on the on each skeleton component (see screenshots):
<img width="1395" alt="search" src="https://user-images.githubusercontent.com/21676914/66067866-1c640600-e512-11e9-9b93-c5c5b87dda0a.png">
